app/src/main/java/info/nightscout/android/medtronic/MainActivity.java

```diff
@@ -93,10 +94,24 @@
     private int chartZoom = 3;
     private boolean hasZoomedChart = false;
 
-    private NumberFormat sgvFormatter;
-    private boolean mmolxl;
+    public static NumberFormat sgvFormatter;
+    public static boolean mmolxl;
     private boolean mmolxlDecimals;
 
+    public static byte radioChannelInuse = 0;
```

Comment:
Before integrating this and the new poll code more fully that it may be worth seeing what is being done with pump history reading and also sending data to NS (pump status, finger bs, iob etc) even when there is no sensor in use.